core/lib/core_web/controllers/layouts/website/html.ex

```diff
@@ -51,8 +52,9 @@ defmodule CoreWeb.Layouts.Website.Html do
                       <%= render_slot(@inner_block) %>
                       <Margin.y id={:page_footer_top} />
                     </div>
-                    <div class="w-0 md:w-sidepadding">
-                    </div>
+                    <%= if @include_right_sidepadding == true do %>
```

Comment:
FYI 
'== true' is hier niet nodig. 
In Elixir mag je ook nog een '?' aan het einde van de variabele plaatsen om een boolean aan te geven